lib/app-components/addon/components/branded-navbar/styles.scss

```diff
@@ -63,8 +63,8 @@
     background-position-y: center;
     background-size: contain;
     background-repeat: no-repeat;
-    // This is overwritten by theme-styles component,
-    // but if there is no provider.asset.square_color_transparent, we fallback to use the logo off the brand
+    // This is overwritten by theme-styles component for Collections,
```

Comment:
This could be written more succinctly with: 

// Preprints uses the brand logo
// Collections uses the theme-styles component